evap/staff/templates/staff_course_review.html

```diff
@@ -27,9 +27,6 @@ <h3 class="panel-title">
                     </h3>
                 </div>
                 <div class="panel-body">
-                    {% for hidden in form.hidden_fields %}
-                        {{ hidden }}
-                    {% endfor %}
```

Comment:
what about the other forms where we render hidden like this, should they be changed as well?